### DISCUSSION TITLE
High-frequency Noise After Running Auto-Editor

### DISCUSSION THREAD

**Discussion by Cust0mfirmware** [WyattBlue/auto-editor#507] [4. Bugs]
### Bug description

For the output files, the audio quality changes - also in places where no speed increase has been applied, high frequency noise ( short high-pitched noise that are not in the input file) can be heard.

### What's your platform?

```shell
OS: Linux 5.15.0-117-generic x86_64
Python: 3.10.12
PyAV: 12.3.0
Auto-Editor: 24w31a
```

### What command did you use

```shell
auto-editor file.mp4 --silent-speed 35 --video-speed 1
```

### What properties does your input video have?

1)
 - recommendedTimebase: 30/1
 - video:
   - track 0:
     - codec: h264
     - fps: 30
     - resolution: 1920x1080
     - aspect ratio: 16:9
     - pixel aspect ratio: 1
     - duration: 3.7
     - pix fmt: yuv420p
     - color range: 1 (tv)
     - color space: 1 (bt709)
     - color primaries: 1 (bt709)
     - color transfer: 1 (bt709)
     - timebase: 1/15360
     - bitrate: 173876
     - lang: und
 - audio:
   - track 0:
     - codec: aac
     - samplerate: 48000
     - channels: 2
     - duration: 3.62
     - bitrate: 255682
     - lang: und
 - container:
   - duration: 3.7
   - bitrate: 437351

2)
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf60.20.100
  Duration: 00:00:03.77, start: 0.000000, bitrate: 429 kb/s
  Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(tv, bt709), 1920x1080 [SAR 1:1 DAR 16:9], 173 kb/s, 30 fps, 30 tbr, 15360 tbn, 60 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
  Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 255 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      vendor_id       : [0][0][0][0]

### Comments

Many thanks for the great tool.

I would also have no problem to re-encode the video files before running auto-editor, if you can prevent the noise.

**Comment by Cfomodz** [comment]
The noise is caused by AAC re-encoding at moderate bitrate (255 kb/s). Use higher quality audio settings:

```bash
# Increase audio bitrate to 320k
auto-editor video.mp4 -b:a 320k
```
